2-mandatory/5-reading-list.js

```diff
@@ -15,11 +15,13 @@ You should modify the tests so that they contain the values that correspond to y
 In this style of testing it is typical to write out as strings exactly what you expect your output to be, 
 without using any variables or any logic like loops, template strings or if statements.
 */
-
+// I DO NOT GET THIS
 const books = [];
```

Comment:
For this part, `books` is an array of objects (books). Each book has a title(string), author (string), alreadyRead (boolean) - these are the properties expected in each book. 

Here's an example below.
`const books = [
  { title: "The Hobbit", author: "J.R.R. Tolkien", alreadyRead: true },
  { title: "The Fault in Our Stars", author: "John Green", alreadyRead: false },
  { title: "To Kill a Mockingbird", author: "Harper Lee", alreadyRead: true },
  { title: "The Hunger Games", author: "Suzanne Collins", alreadyRead: false },
  { title: "The Book Theif", author: "Markus Zusak", alreadyRead: true },
];`

This is quite similar to the writers array in 2-mandatory/1-writers.js. The writes array is an array of objects with a structure like this [{} {} {}].  Each writer has a firstName, lastName, occupation, age and alive properties.